.devcontainer/library-scripts/desktop-lite-debian.sh

```diff
@@ -137,6 +155,23 @@ if [ "${INSTALL_NOVNC}" = "true" ] && [ ! -d "/usr/local/novnc" ]; then
     unzip /tmp/websockify-install.zip -d /usr/local/novnc
     ln -s /usr/local/novnc/websockify-${WEBSOCKETIFY_VERSION} /usr/local/novnc/noVNC-${NOVNC_VERSION}/utils/websockify
     rm -f /tmp/websockify-install.zip /tmp/novnc-install.zip
+
+    # noVNC works best with Python 2 right now. Install the right package and use it.
```

Comment:
> As of January 2020, Python 2 has reached End Of Life status, meaning it will receive no further updates or bugfixes, including for security issues. 

see also: https://www.python.org/doc/sunset-python-2/